aws/structure.go

```diff
@@ -3724,6 +3725,47 @@ func flattenWafWebAclRules(ts []*waf.ActivatedRule) []map[string]interface{} {
 	return out
 }
 
+func flattenWorkLinkNetworkConfigResponse(c *worklink.DescribeCompanyNetworkConfigurationOutput) []map[string]interface{} {
+	config := make(map[string]interface{})
+
+	if c == nil {
+		return nil
+	}
+
+	var emptyVpc bool
+	if c.VpcId == nil || *c.VpcId == "" {
+		emptyVpc = true
+	}
+	if len(c.SubnetIds) == 0 && len(c.SecurityGroupIds) == 0 && emptyVpc {
+		return nil
+	}
+
+	config["subnet_ids"] = schema.NewSet(schema.HashString, flattenStringList(c.SubnetIds))
+	config["security_group_ids"] = schema.NewSet(schema.HashString, flattenStringList(c.SecurityGroupIds))
+	if c.VpcId != nil {
```

Comment:
Nit: This `nil` check is extraneous with the usage of `aws.StringValue()`

Same applies below to the other similar usages.

Reply:
Fixed